### PR TITLE
Import aggregated timeseries code

### PR DESCRIPTION
**Pull request by mhidas**
Import code from data-services and incorporate into the package.

### CODE REVIEW

**Comment by mhidas** [comment]
@diodon FYI. Not complete yet.

File: aodntools/timeseries_products/aggregated_timeseries_template.json [hunk 1]
```diff
@@ -301,6 +301,23 @@
             "long_name": "quality flag for sea_water_turbidity",
             "standard_name": "sea_water_turbidity status_flag"
             },
+        "TURBF": {
+            "ancillary_variables": "TURBF_quality_control",
+            "_FillValue": 999999.0,
+            "long_name": "sea_water_turbidity_in_FTU",
+            "standard_name": "sea_water_turbidity_in_FTU",
```
Comment (mhidas) [inline aodntools/timeseries_products/aggregated_timeseries_template.json]: Remove standard_name.

Comment (diodon) [reply aodntools/timeseries_products/aggregated_timeseries_template.json]: done

**Comment by mhidas** [comment]
@diodon This is ready to review/merge. I've added a quick unittest for the product code. It's far from complete, but good enough for now.

@lwgordonimos FYI. I'd like to merge this today. Feel free to take a quick look either before or after that if you have a minute.

**Comment by mhidas** [comment]
BTW there's more refactoring to come, and adding documentation, in other PRs.

**Comment by lwgordonimos** [comment]
FYI, this doesn't work on Python 2 without pinning some libraries, since they have dropped Python 2 support completely at these versions:

```
    'numpy>=1.13.0,<1.17.0',
    'pandas<0.25.0',
    'xarray<0.12.0'
```

Hopefully a very short term pinning!

**Comment by mhidas** [comment]
:+1: Thanks @lwgordonimos I'll add those in my next PR

**Pull request merged**